src/main/java/controller/BusController.java

```diff
@@ -1,28 +1,94 @@
 package controller;
 
 import db.DatabaseHandler;
-
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import entity.Bus;
+import java.sql.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 
 public class BusController {
-	DatabaseHandler databaseHandler = new DatabaseHandler();
-	Statement statement = databaseHandler.createStatement();
 
-	public void listAllBuses() {
+	public int listAllBuses() {
```

Comment:
I don't really get why you have int instead of void for listAllBuses.
Are you still selecting all info about buses?

Reply:
returning int or boolean is a good practice, by this you can check if the method worked or not